queries/thrift/highlights.scm

```diff
@@ -47,22 +47,24 @@
 ; Types
 
 (enum_identifier) @type
+(definition_type) @type
 (exception_identifier) @type
 (exception_param_type) @type
 (field_type) @type
+(param_type) @type
 (type_identifier) @type
 
 ; Attributes
 
 (annotation_identifier) @attribute
+(uri_def) @attribute
```

Comment:
is uri here like an annotation to something?

Reply:
it's used in special namespace defs, like so
```thrift
namespace xsd test (uri = 'http://thrift.apache.org/ns/OneWayTest')
```